OtripleS.Portal.Web/Services/StudentViews/StudentViewService.Exceptions.cs

```diff
@@ -68,6 +70,27 @@ private void TryCatch(ReturningNothingFunction returningNothingFunction)
             }
         }
 
+        private async ValueTask<List<StudentView>> TryCatch(ReturningStudentsViewFunction returningStudentsViewFunction)
+        {
+            try
+            {
+                return await returningStudentsViewFunction();
+            }
+            catch(StudentDependencyException studentDependencyException)
+            {
+                throw CreateAndLogDependencyException(studentDependencyException);
+            }
+            catch(StudentServiceException studentServiceException)
+            {
+                throw CreateAndLogDependencyException(studentServiceException);
+            }
+            catch(Exception serviceException)
+            {
+                var failedStudentDependencyException = new FailedStudentViewServiceException(serviceException);
```

Comment:
Rename the variable name to failedStudentServiceException

Reply:
What about failedStudentViewServiceException?

Reply:
@ysnarafat I think @ShriHumrudha meant `failedStudentViewServiceException`

Reply:
Okay.